v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-single-model/auto-ml-forecasting-backtest-single-model.ipynb

```diff
@@ -590,6 +591,7 @@
     "    step_size=BACKTESTING_PERIOD,\n",
     "    step_number=NUMBER_OF_BACKTESTS,\n",
     "    model_name=model_name,\n",
+    "    forecast_quantiles=[0.10, 0.90],\n",
```

Comment:
Should we set confidence intervals to [0.025, 0.975] as in [orange juice notebook](https://github.com/Azure/azureml-examples/blob/sagoswami/local_quantile/v1/python-sdk/tutorials/automl-with-azureml/forecasting-orange-juice-sales/forecasting_script.py)?

Reply:
Done.